Gruntfile.js

```diff
@@ -42,6 +46,9 @@ module.exports = function (grunt) {
 
         let processFile = function (abspath, filename) {
             let content = grunt.file.read(abspath);
+            if (!content.includes('commit')) {
+                return null
+            }
             let pageIndex;
             // First separate the Front Matter from the content and parse it
             content = content.split("}     \n");
```

Comment:
Why do we split on such an odd string? `}      \n`? Might be nice to comment on why such an odd string with a specific number of spaces.